lib/rbnacl/hash/blake2b_state.rb

```diff
@@ -0,0 +1,17 @@
+# encoding: binary
```

Comment:
Would suggest just defining this inside `blake2b.rb` instead of making a separate file